core/src/main/java/org/axonframework/eventhandling/tokenstore/jdbc/JdbcTokenStore.java

```diff
@@ -59,36 +64,36 @@
     private final Class<?> contentType;
 
     /**
-     * Initialize the JpaTokenStore with given resources. The given {@code claimTimeout} is used to 'steal' any claim
-     * that has not been updated since that amount of time. The token is serialized to a byte array.
+     * Instantiate a {@link JdbcTokenStore} based on the fields contained in the {@link Builder}.
+     * <p>
+     * Will assert that the {@link ConnectionProvider}, {@link Serializer}, {@link TokenSchema}, {@code claimTimeout},
+     * {@code nodeId} and {@code contentType} are not {@code null}, and will throw an {@link AxonConfigurationException}
+     * if any of them is {@code null}.
      *
-     * @param connectionProvider The provider of connections to the underlying database
-     * @param serializer         The serializer to serialize tokens with
+     * @param builder the {@link Builder} used to instantiate a {@link JdbcTokenStore} instance
      */
-    public JdbcTokenStore(ConnectionProvider connectionProvider, Serializer serializer) {
-        this(connectionProvider, serializer, new TokenSchema(), Duration.ofSeconds(10),
-             ManagementFactory.getRuntimeMXBean().getName(), byte[].class);
+    protected JdbcTokenStore(Builder builder) {
+        builder.validate();
+        this.connectionProvider = builder.connectionProvider;
+        this.serializer = builder.serializer;
+        this.schema = builder.schema;
+        this.claimTimeout = builder.claimTimeout;
+        this.nodeId = builder.nodeId;
+        this.contentType = builder.contentType;
     }
 
     /**
-     * Initialize the JpaTokenStore with given resources. The given {@code claimTimeout} is used to 'steal' any claim
-     * that has not been updated since that amount of time.
+     * Instantiate a Builder to be able to create a {@link JdbcTokenStore}.
+     * <p>
+     * The {@code schema} is defaulted to an {@link TokenSchema}, the {@code claimTimeout} to a 10 seconds duration,
+     * {@code nodeId} is defaulted to the {@link ManagementFactory#getRuntimeMXBean#getName} output and the
```

Comment:
nit:
`{@code nodeId} is `
->
`the {@code nodeId} is `

The rest of the items in the sentence start with `the`

Reply:
Added `the`.